jjb/build-images/ci-build-images.yml

```diff
@@ -0,0 +1,19 @@
+---
+- project:
+    name: ci-build-images
+    project-name: ci-build-images
+    project: ci-build-images
+    mvn-settings: ci-build-images-settings
+    github-org: edgexfoundry
```

Comment:
This is a default value (in jjb/defaults.yaml). It's not necessary to define unless you're changing it.

Reply:
All of these are defaults...or just `github-org`

Reply:
Just `github-org`.